### DISCUSSION TITLE
[PWA Mode]: add short_name option inside `package.json`

### DISCUSSION THREAD

**Discussion by tachibana-shin** [quasarframework/quasar#16432] [Ideas / Proposals]
Hi Quasar Team,

I noticed in the source code of quasar is setting `short_name` of `manifest.json` as the `id` field of the app i don't understand why the team decided like that but i believe it was a mistake.
Source: https://github.com/quasarframework/quasar/blob/6f88c71dea42d9e06f24c362ffcb865c745a2686/app-vite/lib/modes/pwa/utils.js#L53

According to the specification, `short_name` is the short name of the application. it's absurd that we put `id` as `short_name`.

**Comment by metalsadman** [comment]
https://quasar.dev/quasar-cli-vite/developing-pwa/configuring-pwa#configuring-manifest-file
